packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.Variant.Example.tsx

```diff
@@ -108,10 +111,21 @@ export class HorizontalBarChartVariantExample extends React.Component<{}, IHBCVa
           onChange={this._onCheckChange}
           styles={{ root: { marginBottom: '20px' } }}
         />
+        <Toggle
+          label="Show chart mode in percentage"
```

Comment:
Rename toggle label to "Show labels as percentage"

Reply:
done